book/website/communication/citable/citable-cite.md

```diff
@@ -58,6 +58,7 @@ When using others software, it is vital to cite and attribute it properly.
 ::::{tab-set}
 :::{tab-item} GitHub
 :sync: github_tab
+
```

Comment:
Is this intended?

Reply:
@arya1302 can you explain why this page has been updated? Or as Danny is asking, if the edit was intended?

Reply:
On second though, I think it's an edit that might have slipped unintentionally in this PR.

Reply:
Yes, this edit slipped into this PR, and it was not intended.

Reply:
```suggestion
```